test/test.js

```diff
@@ -2381,25 +2381,41 @@ describe('Page', function() {
     }));
   });
 
-  describe('Page.setContent', function() {
-    const expectedOutput = '<html><head></head><body><div>hello</div></body></html>';
+  describe('Page/Frame.setContent', function() {
+    const pageExpectedOutput = '<html><head></head><body><div>hello</div></body></html>';
+    const frameExpectedOutput = '<html><head></head><body><div>world</div></body></html>';
     it('should work', SX(async function() {
       await page.setContent('<div>hello</div>');
-      const result = await page.content();
-      expect(result).toBe(expectedOutput);
+      const pageResult = await page.content();
```

Comment:
Ah sorry, I must've been unclear. I meant that there should be no need to change tests at all since `page.setContent` is just a shortcut for the `frame.setContent` (so it already tests the same codepaths).

Reply:
Okay, I'll defer to your judgment. My reasoning was `page.setContent` using `frame.setContent` is an implementation detail and thus without an explicit call to `frame.setContent` there is no guarantee its public API is actually being used and tested.

Reply:
Do you think that the api coverage linter gives insufficient guarantees? The coverage is runtime-based and makes sure that tests call every public method at least once.

Reply:
I don't know much about how it works, which is why I said I'd defer to you :) I've never worked with a project that has an API coverage mechanism like that. Seems reasonable though!